README.md

```diff
@@ -96,11 +98,12 @@ This repo also includes related tools for [running tasks](#kubernetes-run) and [
 
 *Options:*
 
+Refer to `kubernetes-deploy --help` for the authoritative set of options.
+
+- `--template-dir=DIR`: Used to set the deploy directory. Set `$ENVIRONMENT` instead to use `config/deploy/$ENVIRONMENT`.
 - `--bindings=BINDINGS`: Makes additional variables available to your ERB templates. For example, `kubernetes-deploy my-app cluster1 --bindings=color=blue,size=large` will expose `color` and `size`.
-- `--verbose-log-prefix`: Prefixes each log line with `[#{context}][#{namespace}][#{severity}][#{datetime}]` instead of just `[#{severity}][#{datetime}]`. Useful if you're running multiple concurrent deploys in Shipit.
 - `--allow-protected-ns`: Enables you to deploy to `default`, `kube-system` or `kube-public` if you really know what you're doing. Must be used with `--no-prune` (pruning a protected namespace cannot be enabled).
```

Comment:
Maybe we can drop this one too? I bet no too many people will use this tool to deploy their apps to `kube-system`